<launch>
  <!-- Launch in either navigate mode or mapping mode -->
  <arg name="navigate" default="true" />

  <!-- Include spawn_world.launch and set it's arguments -->
  <include file="$(find uml_3d_race)/launch/spawn_world.launch">
    <!-- World file -->
    <arg name="world" value="level1" />
    <!-- Robot spawn position -->
    <arg name="spawn_x" value="0.0" />
    <arg name="spawn_y" value="0.0" />
    <arg name="spawn_yaw" value="0.0" />

    <arg name="goal_x" value="7.33" />
    <arg name="goal_y" value="1.75" />

    <arg name="navigate" value="$(arg navigate)" />
  </include>
</launch>
